src/backends/gitlab.rs

```diff
@@ -556,7 +556,7 @@ fn api_headers(auth_token: &Option<String>) -> Result<header::HeaderMap> {
     if let Some(token) = auth_token {
         headers.insert(
             header::AUTHORIZATION,
-            format!("token {}", token)
+            format!("bearer {}", token)
```

Comment:
Does it accept lowercase "bearer"? Most expect a capital B

Reply:
Yes, Gitlab accept it. 

But you are right, section 2.1 of RFC6750 states that we should capitalize the prefix. So is as you want.